src/main/java/lotto/domain/LottoPrice.java

```diff
@@ -0,0 +1,41 @@
+package lotto.domain;
+
+import java.util.Objects;
+
+public class LottoPrice {
+    private static final int SINGLE_LOTTO_PRICE = 1000;
+
+    private final int lottoPrice;
+
+    public LottoPrice(int purchaseAmount) {
+        validatePurchaseAmount(purchaseAmount);
+        this.lottoPrice = purchaseAmount;
+    }
+
+    private static void validatePurchaseAmount(int purchaseAmount) {
+        if (purchaseAmount < SINGLE_LOTTO_PRICE) {
+            throw new IllegalArgumentException("구입금액은 최소 1000원 이상이어야 합니다.");
+        }
+    }
+
+    public int countLottoPurchased() {
+        return this.lottoPrice / SINGLE_LOTTO_PRICE;
+    }
+
+    public float floatValue() {
+        return this.lottoPrice;
+    }
+
```

Comment:
```suggestion
    public float value() {
        return this.lottoPrice;
    }

```
이미 반환 타입에 어떻게 반환될 것인지 표현이 되기 때문에 메소드 이름에서는 가능하면 반환 타입을 지정해줄 필요는 없습니다. List와 같은 것들도 마찬가지이구요.